package.json

```diff
@@ -52,6 +52,7 @@
     "onCommand:o.showOutput",
     "onCommand:dotnet.restore",
     "onCommand:csharp.downloadDebugger",
+    "onCommand:debugger.pickProcess",
```

Comment:
'debugger.pickProcess' sounds likely to clash with other extensions. Should this be 'csharp.debuggerPickProcess' so something like that?